redhat-openshift/operator/standard-deployment-crd.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: charts.kubecost.com/v1alpha1
+kind: CostAnalyzer
```

Comment:
might make it more obvious as to what it is if we call it kubecost?

Reply:
For now it is technically defined inside the operator. It was named after our helm chart name.